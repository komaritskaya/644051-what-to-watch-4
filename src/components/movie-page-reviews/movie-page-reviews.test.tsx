import React from 'react';
import renderer from 'react-test-renderer';
import MoviePageReviews from './movie-page-reviews';

const comments = [
  {
    id: `1`,
    user: `User`,
    date: new Date(),
    text: `Hey`,
    rate: 1.0,
  }
];

it(`Render Movie Page Reviews`, () => {
  const tree = renderer
    .create(<MoviePageReviews comments={comments} />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
