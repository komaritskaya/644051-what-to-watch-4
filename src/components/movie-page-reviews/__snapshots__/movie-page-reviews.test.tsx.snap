// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Movie Page Reviews 1`] = `
<div
  className="movie-card__reviews movie-card__row"
>
  <div
    className="movie-card__reviews-col"
  />
  <div
    className="movie-card__reviews-col"
  >
    <div
      className="review"
    >
      <blockquote
        className="review__quote"
      >
        <p
          className="review__text"
        >
          Hey
        </p>
        <footer
          className="review__details"
        >
          <cite
            className="review__author"
          >
            User
          </cite>
          <time
            className="review__date"
            dateTime="2016-12-24"
          >
            11 August, 2020
          </time>
        </footer>
      </blockquote>
      <div
        className="review__rating"
      >
        1
      </div>
    </div>
  </div>
</div>
`;
